packages/patterns-react/src/patterns/SimpleLongForm/SimpleLink.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings } from 'carbon-components';
+import { ArrowRight16 } from '@carbon/icons-react';
+const { prefix } = settings;
+
+/**
+ * Simple Link Component
+ *
+ * @param {*} { link: { href, text, target } }
+ * @returns {*} SimpleLink component
+ */
+/**
+ * Simple Link Component
+ *
+ * @param {object} props props object
+ * @param {string} props.href simple link link
+ * @param {string} props.target simple link text
+ * @returns {*} SimpleLink component
+ */
+const SimpleLink = ({ link: { href, target } }) => {
+  return (
+    <a
+      data-autoid={`${prefix}--simplelink`}
```

Comment:
Switch to the dds prefix.

Reply:
updated the code with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/db7c3e6a903c09f8496332f049b3ca59488ad70b)